### DISCUSSION TITLE
Runtime of SpatialFeaturePlot

### DISCUSSION THREAD

**Discussion by lmpmorales48** [satijalab/seurat#7453] [Q&A]
I've been trying to run SpatialFeaturePlot on some mouse genome data and it's been trying to run for about an hour and a half at this point. It threw me a warning about some genes not being in any known directory and has just been idling since that point trying to generate the plot. Does anyone know how long it runs for? And/or is there a way to make it run faster/actually get it to generate the plot?

Here's the warning it throws and here's the code I've got. 
<img width="878" alt="Screenshot 2023-06-15 at 11 17 35 AM" src="https://github.com/satijalab/seurat/assets/79428398/7b60640f-ebbc-411f-a281-5691dc601ced">
<img width="892" alt="Screenshot 2023-06-15 at 10 49 04 AM" src="https://github.com/satijalab/seurat/assets/79428398/908618b1-18d7-4e9e-998e-c7fbe47aa9e9">

**Comment by ggomes77** [comment]
This indeed might take forever because you are trying to plot more than 5000 genes from what I understand. What I do is to do SpatialFeaturePlot for each individual gene. Type the gene name in the "Features".
